app/controllers/analytics/districts_controller.rb

```diff
@@ -32,6 +28,10 @@ def set_facilities
     @facilities = policy_scope(@organization_district.facilities).order(:name)
   end
 
+  def set_cache_key
+    @cache_key = ["view", @organization_district.analytics_cache_key(@from_time, @to_time)]
```

Comment:
Don't need `view` here, the view `cache do` already adds this.

Reply:
Removed `set_cache_key`.